### ISSUE TITLE
Screen capture issue

### ISSUE THREAD

**Issue by mgtroyas** [Swizpig/ZGloom#18]
First of all great port, I just played it from start to end with no issues.

I tried to save some cool looking screenshots plessing screenprint key, but a bmp is saved with only the health overlay, everything else is transparent. Example:
[Imgur](https://imgur.com/4CC7beg)

Intermission graphics are saved correctly, nonetheless.

**Comment by mgtroyas** [comment]
Attaching both examples:
[ZGloom.zip](https://github.com/Swizpig/ZGloom/files/4433516/ZGloom.zip)

**Comment by Swizpig** [comment]
I've noticed this, this is something funky going on with SDL_SaveBMP and 32bit surfaces with alpha channels. Looking, but IrfanView will show them correctly as a workaround.

**Comment by mgtroyas** [comment]
You are right, Irfanview shows all the image, I batch converted to PNG and now all other image viewers can, too.

**Comment by Swizpig** [comment]
Think I've fixed this.
